src/server/pps/server/api_server.go

```diff
@@ -2316,24 +2315,25 @@ func (a *apiServer) InspectPipelineInTransaction(txnCtx *txncontext.TransactionC
 		return nil, errors.New("cannot inspect future pipelines")
 	}
 
-	key, err := ppsutil.FindPipelineSpecCommitInTransaction(txnCtx, a.env.PFSServer, name, "")
+	pipeline.Name = pipelineName
+	commit, err := ppsutil.FindPipelineSpecCommitInTransaction(txnCtx, a.env.PFSServer, pipeline, "")
 	if err != nil {
-		return nil, errors.Wrapf(err, "pipeline not found: couldn't find up to date spec for pipeline %q", name)
+		return nil, errors.Wrapf(err, "pipeline not found: couldn't find up to date spec for pipeline %q/%q", projectName, pipelineName)
```

Comment:
Why are we not using a pipeline proto here?

Reply:
There are a few places below that seem like they should be operating on protos as well.

Reply:
Because when I updated the places project & pipeline names were used together I searched for `%s/%s`, not `%q/%q`. Fixing …